go/vt/wrangler/traffic_switcher.go

```diff
@@ -581,6 +581,13 @@ func (wr *Wrangler) SwitchWrites(ctx context.Context, targetKeyspace, workflowNa
 			return 0, nil, err
 		}
 
+		ts.Logger().Infof("Resetting sequences")
+		if err := sw.resetSequences(ctx); err != nil {
+			ts.Logger().Errorf("resetSequences failed: %v", err)
+			sw.cancelMigration(ctx, sm)
+			return 0, nil, err
+		}
+
```

Comment:
I think this should only call be called when the table is a sequence table.

Reply:
This could avoid calling the new rpc for most of the cases.

Reply:
Good idea: implemented.